server.js

```diff
@@ -3479,11 +3479,17 @@ cache(function(data, match, sendBadge, request) {
   var spec = match[2];  // eg, AFNetworking
   var format = match[3];
   var apiUrl = 'https://trunk.cocoapods.org/api/v1/pods/' + spec + '/specs/latest';
-  var badgeData = getBadgeData('pod', data);
+  var badgeData;
+  if (type === 'v') {
+    badgeData = getBadgeData('pod', data);
+  } else if (type === 'p') {
+    badgeData = getBadgeData('platform', data);
+  } else if (type === 'l') {
+    badgeData = getBadgeData('license', data);
+  }
```

Comment:
How about this?
```js
const typeToLabel = {'v' : 'pod', 'p': 'platform', 'l': 'license'};
const badgeData = getBadgeData(typeToLabel[type], data);
```
Alternatively you can use those if-else statements to set a `label` variable and then run `getBadgeData` once.